spec/tracebacks_spec.lua

```diff
@@ -3,14 +3,15 @@
 -- Please refer to the LICENSE and AUTHORS files for details
 -- SPDX-License-Identifier: MIT
 
-local util = require "misc.util"
+local util = require "spec.util"
 
 local function assert_test(example, expected_content)
-    local cdir = "spec/tracebacks/"..example.."/";
-    local ok, err = util.execute("cd "..cdir.."&& make")
+    local cdir = util.shell_quote("spec/tracebacks/"..example)
+    local ok, err = util.execute(string.format("cd %s && make", cdir))
     assert(ok, err)
 
-    local ok, _, output_content, err_content = util.outputs_of_execute("cd "..cdir.." && pt-run main.lua")
+    local ok, _, output_content, err_content =
+        util.outputs_of_execute(string.format("cd %s && pt-run main.lua", cdir))
```

Comment:
Any particular reason to explicitly call to `string.format`? We are not doing any complex string formatting, shouldn't concatenation suffice and cleaner in this case?

Reply:
I think with string.format it's easier to see the spaces between the words. In the previous version we were forgetting to add spaces before the "&&"!

    "cd "..cdir.."&&"